trace/cloud-trace-demo-app/app/app.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A sample app demonstrating Stackdriver Trace
+"""
+
+import google.auth
+from flask import Flask, render_template, request
+from opencensus.trace import execution_context
+from opencensus.trace.propagation import google_cloud_format
+from opencensus.trace.samplers import AlwaysOnSampler
+from opencensus.ext.stackdriver.trace_exporter import StackdriverExporter
+from opencensus.ext.flask.flask_middleware import FlaskMiddleware
+
+import requests
+import argparse
+import time,random
+
+app = Flask(__name__)
+
+propagator = google_cloud_format.GoogleCloudFormatPropagator()
+
+def createMiddleWare(exporter, sampler):
+  # Configure Tracing
+  middleware = FlaskMiddleware(
+      app,
+      exporter=exporter,
+      propagator=propagator,
+      sampler=sampler)
+  return middleware
+
+@app.route('/')
+def template_test():
+  time.sleep(random.uniform(0,0.5))
```

Comment:
Some comments in the function would be useful - it's not super clear what this function is supposed to be doing.

Why are we sleeping?
What's the significance of config`keyword`?
What's the significance of `endpoint`?

Reply:
Comments added. @dukeng

Reply:
Comments added